ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml

```diff
@@ -128,7 +128,7 @@ tests:
     make setup_envtest
     export KUBEBUILDER_ASSETS="$(${TMP_BIN}/setup-envtest use --use-env -p path)"
     export SELF="make -f /opt/build-harness/Makefile.prow"
-    export SONAR_GO_TEST_ARGS="-coverpkg=./... ./pkg/... ./operator/... ./agent/... ./manager/..."
+    export SONAR_GO_TEST_ARGS="-coverpkg=./... ./pkg/... ./operator/... ./agent/... ./manager/... ./test/integration/..."
```

Comment:
we do not have `./test/integration/...` folder in release-2.11. do not need to modify here.

Reply:
We have added some integration cases in the 2.11

Reply:
oh, sorry, It's been corrected now.